### DISCUSSION TITLE
Block changes on read only Parameters

### DISCUSSION THREAD

**Discussion by stevo-k** [EnviralDesign/UberGui#18] [Ideas]
Hello Lucas,

thanks for sharing this great tool with us. I like it a lot <3

I have recognized UG is changing data that was set to read-only. This can break a reference unintended when scrolling through the panel and single parameters. 

Is it planned to exclude read-only parameters from the edit function by keeping them visible?
Sorry if I have missed overlooked an option to do this. 

Cheers
Sven

**Comment by EnviralDesign** [comment]
Hey @stevo-k , thank you, glad you're finding it useful!

I actually haven't made any logic yet surrounding read only params, I can easily exclude them from the UI - meaning they won't be visible at all. For 4.1 I think I might work in some options for how to display read only params in a non editable way.

I'll see what I can do

**Comment by EnviralDesign** [comment]
@stevo-k pull the latest build of UG, there is now read only support!

**Reply by stevo-k** [reply]
wow :-D

Many thanks Lucas. Works great.
